<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProcessing</class>
 <widget class="QMainWindow" name="BatchProcessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_videosDir"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_videosDir">
          <property name="text">
           <string>Original Videos Dir</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_tmpDir"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_txtFileList">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_masksDir"/>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_tmpDir">
          <property name="text">
           <string>Temporary Dir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_txtFileList">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Individual Files List</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_masksDir">
          <property name="text">
           <string>Masked Videos Dir</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_resultsDir"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_paramFile">
          <property name="text">
           <string>Parameters File</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_resultsDir">
          <property name="text">
           <string>Tracking Results Dir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_txtFileList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_tmpDir">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="comboBox_paramFile">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="4" rowspan="2">
         <widget class="QPushButton" name="pushButton_start">
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="checkBox_isCompress">
          <property name="text">
           <string>Execute compression</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_numMaxProc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Maximum number of processes:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBox_isTrack">
          <property name="text">
           <string>Execute tracking</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_copyUnfinished">
          <property name="text">
           <string>Copy unfinished analysis</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="spinBox_numMaxProc"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_patternExc"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_patternIn"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_patternIn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>File pattern to include:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_patternExc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>File pattern to exclude:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_isCopyVideo">
          <property name="text">
           <string>Copy original videos to tmp dir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
